### DISCUSSION TITLE
Track headers

### DISCUSSION THREAD

**Discussion by kybos** [muse-sequencer/muse#1136] [Team Posts]
With reference to #729

Just experimenting, what about something like that:
![image](https://user-images.githubusercontent.com/10009541/70501543-9d490a80-1b1e-11ea-93bb-7b4567ee0c35.png)
Seems more visually appealing to me than the single-letter abbreviations...

**Comment by terminator356** [comment]
Yes, exactly what I meant. Looks cool. Love the track column thingy.
I think what stopped me was IIRC I was worried about certain contexts or something.
Maybe it was that mute and solo have two states, can't remember. Still, looks cool.
